lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -1410,5 +1410,15 @@ def test_fail_cell_measure_dims(self):
             cm_dims = self.cube.cell_measure_dims(a_cell_measure)
 
 
+class Test_transpose(tests.IrisTest):
+    def setUp(self):
+        self.cube = stock.lat_lon_cube()
+
+    def test_inplace_data_transpose(self):
```

Comment:
The repetition of "transpose" doesn't add anything, and it doesn't mention that we're interested in the laziness. Perhaps just `test_lazy_data`?